tests/BUILD

```diff
@@ -53,6 +52,22 @@ filegroup(
     srcs = glob(["**/*.txt"]),
 )
 
+#
+# Data source for Unicode handling tests
+#
+pkg_files(
+    name = "utf8_files",
+    srcs = [
+        "//tests/testdata/utf8:files",
+    ],
+    # Note: This is temporary. We need to fix a latent bug where
+    # we are using 555 as the default for many things. That was the
+    # Google internal behavior.
```

Comment:
Is there a task for this?

Reply:
Pointed to https://github.com/bazelbuild/rules_pkg/issues/302